serverless.yml

```diff
@@ -714,6 +714,26 @@ functions:
               - name: request.querystring.blueprint_id
               - name: request.header.origin
 
+  node_api_nc_blueprint_code:
+    handler: handlers/node_api.nc_blueprint_source_code
+    maximumRetryAttempts: 0
+    package:
+      patterns:
+        - 'handlers/node_api.py'
+    layers:
+      - { Ref: PythonRequirementsLambdaLayer }
+    events:
+      - http:
+          path: node_api/nc_blueprint_source_code
+          method: get
+          cors: true
+          caching:
+            enabled: true
+            ttlInSeconds: 300
```

Comment:
question(non-blocking): Is there any downside to storing this for longer periods of time? Days, maybe months? As far as I know, blueprint source codes are immutable, except for new breaking versions of the fullnode.

Reply:
The source code might change as long as it maintains compatibility. We could improve a docstring, add optional metadata, etc.